static/js/lib/queries/auth.js

```diff
@@ -19,7 +19,7 @@ const transformAuthResult = (
     flow:         result.flow,
     state:        result.state,
     errors:       result.errors,
-    redirectUrl:  result.redirect_url,
+    redirect_url: result.redirect_url,
```

Comment:
I don't think this change or the one in authTypes.js was necessary and could have side affects in other areas. I think you just need to type the return value on the first change as I originally described.

Reply:
reverted